garage/tf/policies/gaussian_mlp_policy.py

```diff
@@ -4,16 +4,15 @@
 from garage.core import Serializable
```

Comment:
Can you rewrite the rest of the GaussianMLP* primitives to use this model?

Reply:
Sure.